docs/administration/configuration/hashicorp-consul.md

```diff
@@ -0,0 +1,87 @@
+
+![HashiCorp Consul - Key/Value](~@assets/img/key-value-consul.png)
+
+Now, when we return to the Rundeck Login page, we will see the following:
+
+![Rundeck Welcome Message](~@assets/img/rundeck-welcome-message.png)
+
+## Example: Using HashiCorp for a Cluster
+
+When setting up a cluster, you need to specify your key storage and configuration storage to all of the members of the cluster. That would look something like the following for each member of the cluster:
+
+```bash
+# Encryption for key storage
+rundeck.storage.provider.1.type=db
+rundeck.storage.provider.1.path=keys
+rundeck.storage.converter.1.type=jasypt-encryption
+rundeck.storage.converter.1.path=keys
+rundeck.storage.converter.1.config.encryptorType=custom
+rundeck.storage.converter.1.config.password=bompd87463c3fec
+rundeck.storage.converter.1.config.algorithm=PBEWITHSHA256AND128BITAES-CBC-BC
+rundeck.storage.converter.1.config.provider=BC
+# Encryption for project config storage
+rundeck.projectsStorageType=db
+rundeck.config.storage.converter.1.type=jasypt-encryption
+rundeck.config.storage.converter.1.path=projects
+rundeck.config.storage.converter.1.config.encryptorType=custom
+rundeck.config.storage.converter.1.config.password=bompd87463c3fec
+rundeck.config.storage.converter.1.config.algorithm=PBEWITHSHA256AND128BITAES-CBC-BC
+rundeck.config.storage.converter.1.config.provider=BC
+rundeck.config.storage.converter.1.config.loggerName=config.storage
+```
+
+Instead of doing this for each instance, you could just specify the configuration within Consul and it would be applied to all the different members of the cluster. 
+
+To begin, select the "Key/Value" tab in Consul. 
+
+:::warning
+All key-values must be saved in the following directory: `/config/rundeck`. If those folders do not exist yet, create them now. 
+:::
+
+
+![HashiCorp - Create a Key/Value](~@assets/img/hashicorp-create.png)
+
+After confirming that we are in the Rundeck config directory, we can create a new key/value pair. We can do this by clicking "Create," as shown in the picture above. 
+
+
+![HashiCorp - Create a Key/Value](~@assets/img/hashicorp-key-value-create.png)
+
+Now we are going to create our first Key/Value. Since the first property we see in the example storage configuration above is `rundeck.storage.provider.1.type`, we will create that first. We will put the property in the entry box that says "Key or Folder" and we will put the value in the entry box that says "Value", as shown above. After saving that key/value, we will continue down the list, property by property until we have made it through the entire configuration. 
+
+## Starting Rundeck Using Consul
```

Comment:
Is this section done?  Seems to leave a thought hanging

Reply:
I thought so. I didn't think that I should go through all of the different properties that need to be configured because there are like 20 different key value pairs that I would have to create. I can try to refine the wording a bit so it doesn't seem like a cliff hanger